apps/examples/testcase/le_tc/kernel/kernel_tc_main.c

```diff
@@ -100,7 +100,7 @@ int kernel_tc_main(int argc, char *argv[])
 
 #ifdef CONFIG_TC_KERNEL_LIBC_PTHREAD
 #if (!defined CONFIG_PTHREAD_MUTEX_TYPES)
-#error CONFIG_PTHREAD_MUTEX_TYPES is needed for testing LIBC_PTHREAD TC
+	/* #error CONFIG_PTHREAD_MUTEX_TYPES is needed for testing LIBC_PTHREAD TC */
```

Comment:
Yes. Are you agree with those lines are invalid?

And, if tc is omitted by disabling config, we should inform the user.
Like math tc
```
#else
	printf("tc_libc_math will not be executed. CONFIG_HAVE_DOUBLE is not set.\n");
#endif
```